.github/workflows/build-warehouse-image.yml

```diff
@@ -26,25 +26,73 @@ jobs:
     name: Run static checkers
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - run: sudo apt-get install -y libgraphviz-dev graphviz-dev
```

Comment:
Is `sudo` needed for these installs?

Reply:
yes otherwise you get the issue described [here](https://stackoverflow.com/questions/57982945/how-to-apt-get-install-in-a-github-actions-workflow); the containers have a passwordless sudo so it's expected usage